<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateJournal</class>
 <widget class="QMainWindow" name="CreateJournal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>1079</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(191,201,194);
border:none;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="UpperLogoBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="LogoStack">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <widget class="QTextEdit" name="Motto">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>914</width>
          <height>80</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-bottom:3px;
color:white;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ACCOUNTING MADE EASY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>300</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset>
          <activeon>:/images/AppLogo.png</activeon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
       </widget>
       <zorder>pushButton_3</zorder>
       <zorder>Motto</zorder>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(181,191,174);
color:black;

font: 900 10pt &quot;Segoe UI Black&quot;;
</string>
      </property>
      <property name="text">
       <string>Create Journal Entry</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(191,201,194);


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
         </property>
         <property name="text">
          <string>General Journal Number</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ReadJournalNo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
         </property>
         <property name="text">
          <string> Date</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="ReadDate">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(191,201,194);


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="text">
        <string>NOTE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="ReadDescription">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>60</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:gray;
color:black;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>  Enter Journal Entry</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="JournalTable">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    background-color: rgb(194,196,188);     /* Background color of the table */
    gridline-color: #8F8F8F;        /* Gridline color */
    border: 2px solid #A0A0A0;      /* Border around the table */
}

QHeaderView::section {
    background-color: rgb(91,97,97); 
color:white;     /* Header background color */
    padding: 8px;                   /* Header padding */
    font-weight: bold;              /* Header font */
    border: 2px solid #A0A0A0;      /* Border around headers */
}

QTableWidget::item {
    padding: 5px;                   /* Padding for cells */
    background-color: rgb(209,212,203);      /* Default background color for cells */
}

QTableWidget::item:alternate {
    background-color: rgb(233,245,201);      /* Alternating row color */
}

QTableWidget::item:selected {
    background-color: #A0A0FF;      /* Background color when a cell is selected */
    color: white;                   /* Text color when a cell is selected */
}

QTableWidget QScrollBar:vertical {
    background: #F5F5F5;            /* Vertical scrollbar background */
    width: 15px;                    /* Scrollbar width */
}

QTableWidget QScrollBar::handle:vertical {
    background: #B0B0B0;            /* Vertical scrollbar handle color */
    border-radius: 4px;             /* Rounded scrollbar handle */
}
</string>
      </property>
      <property name="rowCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>300</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Debit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Credit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Memo</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(191,201,194);


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>120</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="TotalDebit">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>113</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Total Debit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="TotalCredit">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>60</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>113</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Total Credit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(191,201,194);


</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="Cancel">
          <property name="geometry">
           <rect>
            <x>42</x>
            <y>20</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Save">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>20</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
